staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -508,6 +508,12 @@ type DeploymentStatus struct {
 	// +optional
 	UnavailableReplicas int32 `json:"unavailableReplicas,omitempty" protobuf:"varint,5,opt,name=unavailableReplicas"`
 
+	// Total number of terminating pods (have a non-null .metadata.deletionTimestamp) targeted by this deployment.
```

Comment:
once a pod is terminated, it no longer appears in this count, right? if so, can we describe in the docs what the indication in the API status is that drops a terminating pod out of this count? is the status.podPhase of Succeeded or Failed? something else?